services/credits-service/Makefile

```
@@ -0,0 +1,93 @@
+# Makefile for Credits Service
+
+.PHONY: help install install-dev test test-unit test-integration lint format type-check security-scan clean run docker-build docker-run
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)